test/lib/runtime/questionnaire_action_test.exs

```diff
@@ -1,8 +1,149 @@
-  describe "clean_i18n_quiz" do
+  describe "Ask.Runtime.QuestionnaireExport/1" do
+    setup do
+      empty_step = %{
+        type: "multiple-choice",
+        title: "",
+        store: "",
+        prompt: %{
+          en: %{
+            sms: "",
+            mobileweb: "",
+            ivr: %{
+              text: "",
+              audio_source: "tts"
+            }
+          }
+        },
+        id: "e7590b58-5adb-48d1-a5db-4a118418ea88",
+        choices: []
+      }
+
+      empty_quiz = %Questionnaire{
+        steps: [empty_step],
+        settings: %{},
+        quota_completed_steps: nil,
+        partial_relevant_config: nil,
+        name: nil,
+        modes: [
+          "sms"
+        ],
+        languages: [
+          "en"
+        ],
+        default_language: "en"
+      }
+
+      simple_quiz = %Questionnaire{
+        empty_quiz
+        | name: "My questionnaire title",
+          settings: %{
+            thank_you_message: %{
+              en: %{
+                sms: "My thank you message"
+              }
+            },
+            error_message: %{
+              en: %{
+                sms: "My error message"
+              }
+            }
+          },
+          steps: [
+            %{
+              type: "multiple-choice",
+              title: "My question title",
+              store: "My variable name",
+              prompt: %{
+                en: %{
+                  sms: "My question prompt",
+                  mobileweb: "",
+                  ivr: %{
+                    text: "",
+                    audio_source: "tts"
+                  }
+                }
+              },
+              id: "0b11a399-9b81-4552-a603-7df50d52f991",
+              choices: []
+            }
+          ]
+      }
+
+      {:ok, empty_quiz: empty_quiz, simple_quiz: simple_quiz}
+    end
+
+    test "exports an empty questionnaire", %{empty_quiz: quiz} do
+      quiz_export = QuestionnaireExport.export(quiz)
+
+      manifest =
+        Map.get(quiz_export, :manifest)
+        |> Map.delete(:audio_files)
+
+      assert manifest == %{
+               default_language: "en",
+               languages: ["en"],
+               modes: ["sms"],
+               name: nil,
+               partial_relevant_config: nil,
+               quota_completed_steps: nil,
+               settings: %{},
+               steps: [
+                 %{
+                   choices: [],
+                   id: "e7590b58-5adb-48d1-a5db-4a118418ea88",
+                   prompt: %{en: %{ivr: %{audio_source: "tts", text: ""}, mobileweb: "", sms: ""}},
+                   store: "",
+                   title: "",
+                   type: "multiple-choice"
+                 }
+               ]
+             }
+    end
+
+    test "exports a simple questionnaire", %{simple_quiz: quiz} do
+      quiz_export = QuestionnaireExport.export(quiz)
+
+      manifest =
+        Map.get(quiz_export, :manifest)
+        |> Map.delete(:audio_files)
```

Comment:
why are you deleting this? there shouldn't be any audio files right?

Reply:
This is what I'm deleting. I don't know if it should be there or not.
```
audio_files: #Stream<[enum: #Function<50.40091930/2 in Stream.resource/3>,
   funs: [#Function<46.40091930/1 in Stream.map/2>]]>
```

The complete manifest (without the deletion):
```
%{audio_files: #Stream<[enum: #Function<50.40091930/2 in Stream.resource/3>,
   funs: [#Function<46.40091930/1 in Stream.map/2>]]>, default_language: "en",
  languages: ["en"], modes: ["sms"], name: "My questionnaire title",
  partial_relevant_config: nil, quota_completed_steps: nil,
  settings: %{error_message: %{en: %{sms: "My error message"}},
    thank_you_message: %{en: %{sms: "My thank you message"}}},
  steps: [%{choices: [], id: "0b11a399-9b81-4552-a603-7df50d52f991",
     prompt: %{en: %{ivr: %{audio_source: "tts", text: ""}, mobileweb: "",
         sms: "My question prompt"}}, store: "My variable name",
     title: "My question title", type: "multiple-choice"}]}
 ```

Reply:
🤔  Mmh, it seems to me that the audio_ids is something that we want to be able to test in the future: assert that the correct audio_ids will be included in the zip.

What do you think of changing a little bit the `export` so that it only returns the `ids`, and in the zip (or new intermediate function) we go fetch the audio files and populate the manifest with them?